test/PushNotificationTests/PushNotificationTests.testdef

```diff
@@ -0,0 +1,18 @@
+{
+    "Tests": [
+        {
+            "Description": "Test unpackaged scenarios for PushNotifications",
+            "Filename": "PushNotificationTests.dll",
+            "Parameters": "/name:UnpackagedTests#metadataSet1::VerifyForegroundHandlerFails",
```

Comment:
Whats the part proceeding the #? Is this specifically running only a single test in the suite?

Reply:
We can metadataSet1 is the value assigned by TAEF with conditional properties. This property is SelfContained='true'. MetadataSet0 is SelfContained='false'. Our tests are currently failing some activations due to an existing WinAppSDK issue: https://github.com/microsoft/WindowsAppSDK/issues/3137#issuecomment-1332851404, so for now I disabled our tests.